wire/internal/wire/wire.go

```diff
@@ -172,9 +172,13 @@ func (g *gen) frame() []byte {
 		}
 		sort.Strings(imps)
 		for _, path := range imps {
-			// TODO(light): Omit the local package identifier if it matches
-			// the package name.
-			fmt.Fprintf(&buf, "\t%s %q\n", g.imports[path], path)
+			// Omit the local package identifier if it matches the package name.
+			name := g.prog.Package(path).Pkg.Name()
```

Comment:
I think this might be incorrect for vendored directories (I forget when that prefix strip happens).

Reply:
Yea, you are right; when I set `export GO111MODULE=off` and run `go test --run TestWire/Vendor`, the previous code crashed because `g.prog.Package(path)` returned `nil`.

Now checking for `nil`, and it works in both `GO111MODUDLE` modes.

Reply:
What I'm saying is: the package information is there, but under the vendored import path. If you collect that information before the vendor prefix stripping and put it in the map, then you will never have the `== nil` case.

Reply:
Ah, OK. Done.